docs/section11/11-3.md

```diff
@@ -0,0 +1,61 @@
+# Service_Water_Heating - Rule 11-3
+**Schema Version:** 0.0.36  
+
+**Mandatory Rule:** TRUE
+
+**Rule ID:** 11-3
+
+**Rule Description:** "Where no service water-heating system exists or has been designed and submitted with design documents but the building will have service water-heating loads, a service water-heating system shall be modeled that matches the system type in the baseline building design"
+
+**Rule Assertion:** Options are PASS/FAIL
+
+**Appendix G Section Reference:** Table G3.1 #11, proposed column, c
+
+**Evaluation Context:** P-RMD each SHW bat  
+**Data Lookup:**   
+**Function Call:** 
+- **get_component_by_id**
+- **get_SWH_equipment_associated_with_each_swh_distribution_system**
+
+**Applicability Checks:**
+- check that the SHW BAT in the P_RMD has SHW loads
+
+## Applicability Checks:
+- projects without SWH in the user model are expected to have a baseline SWH system in the proposed model.  If there is SWH in the user model, we assume there is a SWH system designed, and instead rule 11-1 Applies and P_RMD matches U_RMD.
+- use the function get_SWH_equipment_associated_with_each_swh_distribution_system to get a dictionary of SWH equipment associated with each Distribution System for the Proposed model (pumps, tanks, SWH use, distribution system, etc): `p_swh_system_and_equip_dict = get_SWH_equipment_associated_with_each_swh_distribution_system(P_RMD)`
+- for each distribution system: `for distribution_id in p_swh_system_and_equip_dict:`
+    - get the distribution system: `p_distribution = get_component_by_id(P_RMD, distribution_id)`
+    - for each swh use in the dictionary, check to see that the swh use exists in the user model: `for swh_use_id in p_swh_system_and_equip_dict[distribution_id]["SPACE_USES"]:`
+        - get the user swh_use: `u_swh_use = get_component_by_id(U_RMD, swh_use_id)`
+        - if the u_swh_use exists, this rule is not applicable and 11-1 applies: `if(u_swh_use): NOT_APPLICABLE`
```

Comment:
This makes the rule not applicable if even 1 ServiceWaterHeatingUse is defined in the User model. What is supposed to happen if there are just certain spaces where the ServiceWaterHeatingUse is not defined?

Reply:
if this rule is not applicable, rule 11-1 is applicable.  This logic just checks whether the SWH use that exists in the proposed model exists in the user model.  It allows for spaces without SWH uses